content/proposals.md

```diff
@@ -193,7 +206,8 @@ is as follows:
         filename that follows the format `1234-my-new-proposal.md` where
         `1234` is the MSC ID.
     -   Your PR description must include a link to the rendered Markdown
-        document and a summary of the proposal.
+        document and a disclosure of potential conflicts of interest where
+        they might reasonably apply.
```

Comment:
Empirically, proposal summaries in the PR description don't appear to happen at all in practice. Therefore, I've dropped this part when adding the CoI clause.